.github/workflows/rails_tests.yml

```
@@ -193,6 +193,17 @@ jobs:
           aws_region: us-east-1
           flags: --recursive
 
+      - name: 'Fetch 2022 -> 2024 source files'
+        uses: keithweaver/aws-s3-github-action@v1.0.0
+        with:
+          command: cp
+          source: ${{ secrets.AWS_S3_ACTION_BUCKET }}/${{ vars.AWS_S3_ACTIONS_SOURCE_PATH_4_2023 }}
```

I'm guessing you need 
```suggestion
          source: ${{ secrets.AWS_S3_ACTION_BUCKET }}${{ vars.AWS_S3_ACTIONS_SOURCE_PATH_4_2023 }}
```